base/io.jl

```diff
@@ -509,13 +507,17 @@ end
 unsafe_write(s::IO, p::Ptr, n::Integer) = unsafe_write(s, convert(Ptr{UInt8}, p), convert(UInt, n))
 write(s::IO, x::Ref{T}) where {T} = unsafe_write(s, x, Core.sizeof(T))
 write(s::IO, x::Int8) = write(s, reinterpret(UInt8, x))
-function write(s::IO, x::Union{Int16,UInt16,Int32,UInt32,Int64,UInt64,Int128,UInt128,Float16,Float32,Float64})
-    return write(s, Ref(x))
-end
-
 write(s::IO, x::Bool) = write(s, UInt8(x))
 write(to::IO, p::Ptr) = write(to, convert(UInt, p))
 
+function write(io::IO, x::T) where T
+    if isbits(T)
```

Comment:
Shouldn't this be `isbitstype(T)` ?  Alternative: `isbits(x)` but it looks like this would reduce performance.

Reply:
Good catch, I did not follow up on the API update.